Makefile

```diff
@@ -111,6 +112,18 @@ ifeq "$(LXD_OFFLINE)" ""
 endif
 	swagger generate spec -o doc/rest-api.yaml -w ./lxd -m
 
+.PHONY: doc
+doc:
+	@echo "Setting up documentation build environment"
```

Comment:
Can we add a separate target for only building the documentation? When working on the docs, I constantly rebuild, and I don't want to wait for the dependencies to download and install every time.